ivy_tests/test_ivy/test_frontends/test_tensorflow/test_raw_ops.py

```diff
@@ -1062,3 +1062,30 @@ def test_tensorflow_zeros_like(
         fn_tree="raw_ops.ZerosLike",
         x=np.asarray(x, dtype=dtype),
     )
+
+
+# Relu
+@handle_cmd_line_args
+@given(
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("numeric"), min_num_dims=10, max_num_dims=20
```

Comment:
Are we sure that Relu needs a min of 10 dimensions for testing?

Reply:
I am sorry, I meant to write 1. I've changed it to 1 now. Thanks.